File: 1561.Maximum-Number-of-Coins-You-Can-Get/Maximum-Number-of-Coins-You-Can-Get.cpp
class Solution {
public:
    int maxCoins(vector<int>& piles) {
    sort(piles.begin(),piles.end());
      int ans = 0;
        for (int i =0, j= piles.size() -2; i < j; i++, j-=2)
        ans += piles[j];
        return ans;
    }
};

File: leetcode-solutions/maximum-number-of-coins-you-can-get.py
class Solution:
    def maxCoins(self, piles: List[int]) -> int:
        if len(piles)==3: return sorted(piles)[1] 
        else:
            arr,ans = sorted(piles),0
            j=len(piles)-2
            i=0
            while i<len(piles)//3:
                ans+=arr[j]
                j-=2
                i+=1
            return ans
